src/pages/HomePage.tsx

```diff
@@ -140,16 +140,14 @@ const Login = ({ walletNames, onLinkClick }: LoginProps) => {
           {t`Login`}
         </Button>
       </SectionStyled>
-      <SwitchLink onClick={onLinkClick} centered>
-        {t`Create / import a new wallet`}
-      </SwitchLink>
+      <SwitchLink onClick={onLinkClick}>{t`Create / import a new wallet`}</SwitchLink>
     </>
   )
 }
 
 const InitialActions = ({
   showLinkToExistingWallets,
-  onLinkClick
+  onLinkClick = () => ({})
```

Comment:
Why this default?

Reply:
So it's a no-op if nothing is passed and `SwitchLink` will still work if `showLinkToExistingWallets` is true